core/services/keeper/registry_interface.go

```diff
@@ -209,11 +225,14 @@ func (rw *RegistryWrapper) GetConfig(opts *bind.CallOpts) (*RegistryConfig, erro
 	case RegistryVersion_1_0, RegistryVersion_1_1:
 		config, err := rw.contract1_1.GetConfig(opts)
 		if err != nil {
-			return nil, errors.Wrap(err, "failed to get contract config")
+			// TODO: error wrapping with %w should be done here to preserve the error type as it bubbles up
```

Comment:
Could you create a ticket for it, if not already done :)

Reply:
Absolutely!